library-core/src/main/java/com/mikepenz/fastadapter/listeners/CustomEventHook.kt

```diff
@@ -21,12 +20,8 @@ abstract class CustomEventHook<Item : GenericItem> : EventHook<Item> {
      * @param viewHolder
      * @return
      */
-    fun getFastAdapter(viewHolder: RecyclerView.ViewHolder): FastAdapter<Item>? {
-        val tag = viewHolder.itemView.getTag(R.id.fastadapter_item_adapter)
-        return if (tag is FastAdapter<*>) {
```

Comment:
I see these types of casts a lot. As far as I'm aware, Java/Kotlin can't guarantee generic types anyways, so is there a need to check that something is an instance of `FastAdapter<*>` before checking if it's an instance of `FastAdapter<Item>`? Or are the same thing?

Likewise, if they are interchangeable, casting to `FastAdapter<Item>` removes further checks per item to see if they are actually of type `Item` vs `GenericItem`